### ISSUE TITLE
No Initialization of masks in train_n.py 

### ISSUE THREAD

**Issue by rengchizder** [gu18168/HierGAT#2]
When running the code for HierGAT+, an error is thrown. I believe it is because there is no initialization of masks in train_n.py line 38: logits, y, _ = model(xs, zs, y, masks). 

Thanks.
